circuit-lib/circuit-lib.circom/benchmarks/index.ts

```diff
@@ -0,0 +1,319 @@
+const bench = require("micro-bmark");
+const { compare, run } = bench;
+const snarkjs = require("snarkjs");
```

Comment:
This is an unexpected location for this code.
Please move to a utils file in the light directory and ideally find a better name for it.
(I think the light directory is the right place for this logic because it contains a lot of code which will only be used for light protocol circuit benchmarks.)

Reply:
renamed it as utils and moved it inside ./benchmarks/light